app/assets/stylesheets/application.css.scss

```diff
@@ -255,8 +278,24 @@ main {
     top: 9px;
     right: 2px;
     position: absolute;
+    border-radius: 5px;
+    color: white;
+    background-color: black;
+    border: none;
+}
+
+.avatar-selection{
+font-size: 13px
 }
+// Summary page
 
+.item_name {
+  color: green;
```

Comment:
Color `green` should be written in hexadecimal form as `#008000`<br>Color literals like `green` should only be used in variable declarations; they should be referred to via variable everywhere else.